numpy/polynomial/_polybase.py

```diff
@@ -123,17 +123,16 @@ def _line(self):
         pass
 
     @abstractmethod
-    def _interp(self):
-        pass
-
-    @abstractmethod
     def _roots(self):
         pass
 
     @abstractmethod
     def _fromroots(self):
         pass
 
+    def _interp(self):
```

Comment:
The new functionality is pretty much Chebyshev specific, so I wouldn't add anything to the base class, but rather to the derived Chebyshev class at the end of the `chebyshev.py` file.